### PR TITLE
:sparkles: Adds Ubuntu 2004 CIS to the repo

### PR DESCRIPTION
**Pull request by mm-weber**
adds - Ubuntu 20.04 CIS Cnspec to the repo

### CODE REVIEW

File: aws/ec2-instances/amis.tf [hunk 1]
```diff
@@ -145,6 +145,22 @@ data "aws_ami" "ubuntu2004" {
   owners = ["099720109477"]
 }
 
+data "aws_ami" "ubuntu2004_cis_cnspec" {
```
Comment (atomic111) [inline aws/ec2-instances/amis.tf:148]:
```suggestion
data "aws_ami" "ubuntu2004_cis" {
```

File: aws/ec2-instances/main.tf [hunk 2]
```diff
@@ -514,6 +514,56 @@ module "rhel8_cis_cnspec" {
...
+  user_data_replace_on_change = true
+}
+
+module "ubuntu2004_cis_cnspec" {
+  source  = "terraform-aws-modules/ec2-instance/aws"
+  version = "~> 5.2.1"
+
+  create                      = var.create_ubuntu2004_cis_cnspec
+  name                        = "${var.prefix}-ubuntu2004-cis-cnspec-${random_id.instance_id.id}"
+  ami                         = data.aws_ami.ubuntu2004_cis_cnspec.id
```
Comment (atomic111) [inline aws/ec2-instances/main.tf:555]:
```suggestion
  ami                         = data.aws_ami.ubuntu2004_cis.id
```

**Comment by atomic111** [comment]
@mm-weber thanks for adding ubuntu 20.04. Can you please add also `ubuntu2004_cis` to the main.tf then we get the following variants:

- ubuntu2004
- ubuntu2004_cnspec
- ubuntu2004_cis
- ubuntu2004_cis_cnspec

File: aws/ec2-instances/amis.tf (hunk 1 above)
Comment (mm-weber) [reply aws/ec2-instances/amis.tf:148]: fixed

File: aws/ec2-instances/main.tf (hunk 2 above)
Comment (mm-weber) [reply aws/ec2-instances/main.tf:555]: fixed

**Comment by mm-weber** [comment]
> @mm-weber thanks for adding ubuntu 20.04. Can you please add also `ubuntu2004_cis` to the main.tf then we get the following variants:
> 
> * ubuntu2004
> * ubuntu2004_cnspec
> * ubuntu2004_cis
> * ubuntu2004_cis_cnspec

Done

File: aws/ec2-instances/outputs.tf [hunk 3]
```diff
@@ -54,6 +54,21 @@ output "rhel9_cnspec" {
...
+}
+
+output "ubuntu2004_cnspec" {
+  value = module.ubuntu2004_cnspec.public_ip == null ? "" : "ssh -o StrictHostKeyChecking=no -i ~/.ssh/${var.aws_key_pair_name} ubuntu@${module.ubuntu2004_cnspec.public_ip}"
+}
+
+output "ubuntu2004_cis_cnspec" {
+  value = module.ubuntu2004_cis_cnspec.public_ip == null ? "" : "ssh -o StrictHostKeyChecking=no -i ~/.ssh/${var.aws_key_pair_name} ubuntu@${module.ubuntu2004_cis_cnspec.public_ip}"
+}
+
```
Comment (atomic111) [inline aws/ec2-instances/outputs.tf:69]: ubuntu2004_cis is missing in the outputs

Comment (mm-weber) [reply aws/ec2-instances/outputs.tf:74]: fixed

**Review by atomic111 — APPROVED** [review]
@mm-weber thank you for the great improvement

**Pull request merged**